### DISCUSSION TITLE
Language localisation

### DISCUSSION THREAD

**Discussion by tfboy** [netbox-community/netbox#11108] [Q&A]
I've searched and couldn't find this asked before so apologies if it has and I've missed it.
But is there any way to offer Netbox in a different language? Is English the only option?
It would seem a simple thing to add - have a dictionary lookup file with all the text translations into the local language.
Back-end would of course remain in English, but all text presented through the web interface could be localised.
I'd be happy to contribute for French...
And I fully understand that if some words are longer, it may make a mess of formatting...

**Comment by LLFT** [comment]
Hello,

The product is indeed very promising, but the localization would be a small plus not negligible.

Do you have that option?

Thank you for returning.

**Comment by candlerb** [comment]
> But is there any way to offer Netbox in a different language? Is English the only option?

As of v3.4 there is an experimental [localization feature](https://docs.netbox.dev/en/stable/configuration/system/#enable_localization) which will set the date format, and also can result in strings from some third-party libraries being translated (e.g. the word for "Tags" may be replaced), but Netbox does not currently have localization for its own text.

This setting also breaks some functionality - see #11271
